components/dashboard/src/menu/Menu.tsx

```diff
@@ -98,31 +106,38 @@ export default function Menu() {
                                         />
                                     </li>
                                 )}
-                                {isGitpodIo && (
-                                    <li className="cursor-pointer">
-                                        <PillMenuItem name="Feedback" onClick={handleFeedbackFormClick} />
+                                {!isDataOps && (
+                                    <li>
+                                        <a
+                                            href="/"
+                                            className="flex items-center gap-x-1 text-sm text-gray-500 dark:text-gray-400 hover:text-gray-900 dark:hover:text-white"
+                                        >
```

Comment:
we should be able to use `text-pk-content-secondary` here to not have to specify the dark theme colors. Applies to the mobile rendition below as well.

Reply:
Resolved in 657d73f